.stylelintrc.yml

```diff
@@ -51,6 +51,66 @@ rules:
     - ignorePattern: /--rh-font-family-/
 
   rhds/token-values: true
+  rhds/no-unknown-token-value:
```

Comment:
@bennypowers, I've updated the token version on package.json and package-lock.json. Am I getting a bunch of errors because tokens 2.0 hasn't been released yet?